lib/PhpParser/PrettyPrinter/Standard.php

```diff
@@ -197,13 +197,18 @@ protected function pScalar_InterpolatedString(Scalar\InterpolatedString $node):
     }
 
     protected function pScalar_Int(Scalar\Int_ $node): string {
+        if ($node->getAttribute('shouldPrintRawValue') === true) {
+            return (string) $node->getAttribute('rawValue');
```

Comment:
```suggestion
            return $node->getAttribute('rawValue');
```
`rawValue` should already be a string -- omitting the cast will result in a TypeError if it has some unexpected type (e.g. null because the attribute is missing).